make/harbor.yml.tmpl

```diff
@@ -199,3 +199,30 @@ proxy:
 #   enabled: false
 #   port: 9090
 #   path: /metrics
+
+# Trace related config
+# only can enable one trace provider(jaeger or otel) at the same time,
+# and when using jaeger as provider, can only enable it with agent mode or collector mode.
+# if using jaeger collector mode, uncomment endpoint and uncomment username, password if needed
+# if using jaeger agetn mode uncomment agent_host and agent_port
+# trace:
+#   enabled: true
+#   # set sample_rate to 1 if you wanna sampling 100% of trace data; set 0.5 if you wanna sampling 50% of trace data, and so forth
+#   sample_rate: 1
+#   # # namespace used to diferenciate different harbor services
+#   # namespace:
+#   # # attributes is a key value dict contains user defined attributes used to initialize trace provider
+#   # attributes:
+#   #   application: harbor
+#   # jaeger:
```

Comment:
When we parse these settings, the logic is either endpoint or agent, but uses the length as the judgment. 

Most likely is, user uncomment jadge, but let the endpoint be the default value(http://hostname:14268/api/traces). But, the code goes into the endpoint path, right?